finetuner-workflow/finetuner/ds_config.json

```diff
@@ -32,12 +32,14 @@
         "reduce_scatter": true,
         "reduce_bucket_size": 2e8,
         "contiguous_gradients": true,
-        "cpu_offload": true,
-        "stage3_gather_fp16_weights_on_model_save": true
+        "offload_optimizer": {
+            "device": "cpu"
+        },
+        "stage3_gather_16bit_weights_on_model_save": true
     },
     "gradient_accumulation_steps": "auto",
     "gradient_clipping": "auto",
-    "steps_per_print": 2000,
+    "steps_per_print": 1000000000000000,
```

Comment:
This looks odd to me. Is there an explanation why it went to this value from 2000?

Reply:
It doesn't allow disabling the logging, so we set it to an extremely high number.